### ISSUE TITLE
cmb.py module uses incorrect API for so_pysm_models/alms.py

### ISSUE THREAD

**Issue by msyriac** [simonsobs/mapsims#24]
@zonca  Currently, the cmb.py module cannot be used to generate lensed CMB sims, since it uses an API for so_pysm_models/alms.py that seems outdated.

e.g.:
```
ch = Channel('LA',145)                                                                                                                                                         
cmbsim = SOStandalonePrecomputedCMB(inum, nside=nside, lensed=True, aberrated=False, has_polarization=True, cmb_set=cmb_set, input_units='uK_CMB')
hmap = cmbsim.simulate(ch=ch,output_units='uK_CMB') 

Traceback (most recent call last):
  File "demo.py", line 37, in <module>
    hmap = cmbsim.simulate(ch=ch,output_units='uK_CMB')
  File "/global/homes/m/msyriac/repos/mapsims/mapsims/cmb.py", line 145, in simulate
    output_units=output_units,
TypeError: get_emission() got an unexpected keyword argument 'nu'
```

Was this inconsistency introduced recently? Isn't the *PrecomputedCMB family of classes essential for sim generation?

See also : https://github.com/simonsobs/so_pysm_models/issues/42 and https://github.com/simonsobs/so_pysm_models/pull/43

**Comment by zonca** [comment]
Please use a older release. I am breaking api and outdated docs on master.
I'll fix after we finish TOD sims and have time available.

**Comment by zonca** [comment]
this is fixed in 2.0.0 and above
